src/MySqlEventStore.php

```diff
@@ -490,13 +499,59 @@ public function getDefaultReadModelProjectionFactory(): ReadModelProjectionFacto
         if (null === $this->defaultReadModelProjectionFactory) {
             $this->defaultReadModelProjectionFactory = new PdoEventStoreReadModelProjectionFactory(
                 $this->connection,
-                $this->eventStreamsTable
+                $this->eventStreamsTable,
+                $this->projectionsTable
             );
         }
 
         return $this->defaultReadModelProjectionFactory;
     }
 
+    public function deleteProjection(string $name, bool $deleteEmittedEvents): void
+    {
+        $sql = <<<EOT
+UPDATE $this->projectionsTable SET status = ? WHERE name = ? LIMIT 1;
+EOT;
```

Comment:
Postgres has not UPDATE ... LIMIT. But we can remove it from MySqlEventStore for consistence reasons, anyway, there no possibility to update more than one row, as name is unique.